Makefile

```diff
@@ -128,7 +131,11 @@ firmwares: stamp-clean-firmwares .stamp-firmwares
 	rm -rf $(IB_BUILD_DIR)
 	mkdir -p $(IB_BUILD_DIR)
 	$(eval TOOLCHAIN_PATH := $(shell printf "%s:" $(OPENWRT_DIR)/staging_dir/toolchain-*/bin))
+ifeq ($(BUILDTYPE),unstable)
```

Comment:
please suggest a regex covering the normal filename and also the filename with the version-string included.